Remove Travis CI Integration (#313)
acquia/orca: docs/getting-started.md

# Getting Started

> 1. [Configuring GitHub Actions](#configuring-github-actions)

Reviewer: ```suggestion
1. [Configuring CI](#configuring-ci)
```

See comment on line 11.

1. [Local installation](#local-installation)
1. [Making ORCA aware of your package](#making-orca-aware-of-your-package)
1. [Running automated tests](#running-automated-tests)
    1. [PHPUnit](#phpunit)
    1. [Nightwatch.js](#nightwatchjs)
    1. [Tagging/grouping](#tagginggrouping)

> ## Configuring GitHub Actions

Reviewer: Let's keep this generic--not product-specific--to minimize possible changes needed in the future.

```suggestion
## Configuring CI
```

> ORCA's primary use case is in a continuous integration workflow, running against pull requests and commits. It provides several scripts in `bin/common` corresponding to build phases and steps in GitHub Actions, respectively:

Author: Done

# Getting Started

1. [Configuring CI](#configuring-ci)
1. [Local installation](#local-installation)
1. [Making ORCA aware of your package](#making-orca-aware-of-your-package)
1. [Running automated tests](#running-automated-tests)
    1. [PHPUnit](#phpunit)
    1. [Nightwatch.js](#nightwatchjs)
    1. [Tagging/grouping](#tagginggrouping)

## Configuring CI

> ORCA's primary use case is in a continuous integration workflow, running against pull requests and commits. It provides several scripts in `bin/ci` corresponding to build phases and steps in GitHub Actions, respectively:

Reviewer: ```suggestion
ORCA's primary use case is in a continuous integration workflow, running against pull requests and commits. It provides several scripts in `bin/ci` corresponding to build phases and steps in GitHub Actions:
```

Error: Invalid value for "--sut" option: "drupal/example".
```

> To make ORCA aware of your package you'll need to dynamically add it to the list using [environment variables](advanced-usage.md#ORCA_PACKAGES_CONFIG_ALTER). Doing this on CI is covered in the `env.global` section of the [example CI configuration](../example/.github/workflows/orca.yml). Locally, you must set the appropriate variable(s) in your terminal session. The assignments can be copied right from your `orca.yml`. Just prefix them with the `export` command, e.g.:

Reviewer: The schema location is different in GitHub Actions. You might do a quick search to make sure we don't refer to similar Travis configuration paths.

```suggestion
To make ORCA aware of your package you'll need to dynamically add it to the list using [environment variables](advanced-usage.md#ORCA_PACKAGES_CONFIG_ALTER). Doing this on CI is covered in the `jobs.build.env` section of the [example CI configuration](../example/.github/workflows/orca.yml). Locally, you must set the appropriate variable(s) in your terminal session. The assignments can be copied right from your `orca.yml`. Just prefix them with the `export` command, e.g.:
```
